Hazel/src/Hazel/Core.h

```diff
@@ -1,27 +1,37 @@
 #pragma once
 
-#ifdef HZ_PLATFORM_WINDOWS
+#if defined HZ_PLATFORM_WINDOWS
+        #define HZ_DEBUG_BREAK { __debugbreak(); }
```

Comment:
Wouldn't it be cleaner to use function macros?
```suggestion
        #define HZ_DEBUG_BREAK() __debugbreak();
```